src/config/rcxml.c

```diff
@@ -863,8 +869,11 @@ entry(xmlNode *node, char *nodename, char *content)
 		} else {
 			wlr_log(WLR_ERROR, "Invalid value for <resize popupShow />");
 		}
+	} else if (!strcasecmp(nodename, "device.touch")) {
```

Comment:
I think we should avoid the term `device` here because it's used in `xml_tree_walk()` which makes it special.
https://github.com/labwc/labwc/blob/4238d7fc33dc2c7c4ee4ecbac81decbc91833c1a/src/config/rcxml.c#L948

Maybe `device_name`?